sources/plugins/AB-Why3/index_abwhy3.mld

```diff
@@ -0,0 +1,75 @@
+{1 ABWhy3 plugin}
+
+{2 What is this plugin ?}
+
+An experimental front-end that parses a subset of Why3's logic. More
+precisely, this front-end targets proof obligations generated by the
+Atelier-B framework in Why3 format. It should be used with a prelude
+defining the B Set theory (currently provided in
+[preludes/b-set-theory-prelude-2020-02-28.ae]).
+
+
+{2 What this plugin is not ?}
+
+This plugin mainly focuses on the shape of the proof obligations
+produced by the Why3 proofs obligations generator of Atelier-B. You
+will probably not be able to use it to parse/solve other formulas
+written in Why3's logic.
+
+
+
+{2 How to use it ?}
+
+Assuming you are currently in [alt-ergo-git/sources] directory, and
+[autoconf && ./configure && make && make AB-why3] succeeded, you can ask Alt-Ergo to
```

Comment:
I don't have the possibility to autoconf, the configure is already present in my sources directory. 
```suggestion
[./configure && make && make AB-Why3] succeeded, you can ask Alt-Ergo to
```